### PR TITLE
fix: (core) Add Additional class to popover component, introduce nested popover example

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/1633
#### Please provide a brief summary of this pull request.
There is added new property on popover, to add custom classlist for popover container. It was missing mostly, when user wanted to put popover inside popover, to differentiate them.
I added also interesting use case with `nested-popover`, as example.
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
To be honest I don't know how I feel about this nested popover...
In terms of code it looks good. About the look... I think we need some design before releasing this. 
@droshev what do you think about this?

Screen Shot 2019-11-25 at 2 54 24 PM

**Comment by JKMarkowski** [comment]
Hi @InnaAtanasova, this was more to show how user can use class names injection.

**Comment by droshev** [comment]
I agree with @InnaAtanasova. It looks odd the way it appears.

**Comment by JKMarkowski** [comment]
Nested class example is removed.

File: apps/docs/src/app/core/core-documentation.module.ts [hunk 1]
```diff
@@ -756,7 +756,7 @@ import { SelectTypesExampleComponent } from './component-docs/select/examples/se
         NotificationContentComponent,
         DatePickerComplexI18nExampleComponent,
         DatetimePickerComplexI18nExampleComponent,
-        PopoverDropdownComponent
+        PopoverDropdownComponent,
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/core-documentation.module.ts]: unnecessary comma

**Pull request merged**